src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const lines = sourceString.split(';').filter((line) => line.trim() !== '');
```

Comment:
```suggestion
  const lines = sourceString.split(';').filter((line) => line.trim());
```
You don't need to compare with an empty string, you can just check if value exists
https://developer.mozilla.org/en-US/docs/Glossary/Falsy